lib/crowbar/client/config.rb

```diff
@@ -164,7 +164,12 @@ def default_apiversion
         if ENV["CROWBAR_APIVERSION"].present?
           ENV["CROWBAR_APIVERSION"].to_f
         else
-          1.0
+          os_release = Hash[File.open("/etc/os-release").read.scan(/(\S+)\s*=\s*"([^"]+)/)]
+          if os_release["VERSION_ID"] < "12.2"
```

Comment:
maybe return `1.0` only if `os_release["VERSION_ID"]` is `12.1`, and `"ID"=>"sles"`.  in all other cases `2.0`